EIPS/EIP-8.md

```diff
@@ -0,0 +1,148 @@
+### Title
+
+      EIP: 8
+      Title: devp2p Forward Compatibility Requirements for Homestead
+      Author: Felix Lange <felix@ethdev.com>
+      Status: Draft
+      Type: Standards Track
+      Layer: Networking
+      Created: 2015-12-18
+
+### Abstract
+
+This EIP introduces new forward-compatibility requirements for implementations of the
+devp2p Wire Protocol, the RLPx Discovery Protocol and the RLPx TCP Transport Protocol.
+
+### Specification
+
+Implementations of the devp2p Wire Protocol should ignore the version number of hello
+packets. If the remote hello packet contains a version greater than the current local
+version, the implementation should reply with a hello packet containing the local (lower)
+version. Implementations should also ignore any additional list elements at the end of the
+hello packet.
```

Comment:
Aren't the hello packets being sent out immediately as the connection is established? Also not sure about it, but AFAIK there are no ordering guarantees between who sends such a packet first or second, so I don't really see a way to send the lower of the two, if both sides have to wait for the remote one to figure out the correct version.